### DISCUSSION TITLE
I want to Use the selenium module, but the selenium module requires a chromedriver file in the local path to operate

### DISCUSSION THREAD

**Discussion by skeletonwarrior** [microsoft/autogen#889] [Q&A]
When executing code through docker with autogen, using the selenium module seems very powerful, but the selenium module requires a chromedriver file in the local path to operate. To run selenium through docker in autogen normally, how should I set up chromedriver?

**Comment by rickyloynd-microsoft** [comment]
You may also want to ask this on our [Discord channel](https://discord.gg/pAbnFJrkgZ).

**Comment by martabrucka** [comment]
I am facing the same problem

**Comment by heijligers** [comment]
Bump

**Comment by ekzhu** [comment]
If you want to supply your own docker image with batteries included, currently you can set `code_execution_config={"user_docker": "your image name"}`. You would need to update the `AssistantAgent`'s system prompt to make it aware of the tools it has.

**Reply by martabrucka** [reply]
should be: "use_docker"
